src/platform_impl/web/web_sys/canvas.rs

```diff
@@ -158,97 +185,147 @@ impl Canvas {
         // The `keypress` event is deprecated, but there does not seem to be a
         // viable/compatible alternative as of now. `beforeinput` is still widely
         // unsupported.
-        self.on_received_character = Some(self.add_user_event(
+        self.on_received_character = Some(self.add_event(
```

Comment:
Why change from 'add_user_event' to 'add_event'?

Reply:
Oups! A mistake...